groceristar.js

```diff
@@ -26,17 +27,15 @@ module.getDepartments = function() {
 }
 
 module.getAllDepartments = function(){
```

Comment:
I'm not sure if this function works. I think you delete something important by mistake here

Reply:
Or grocery is coming from top. if yes, let's just ignore that